quill-core/src/test/scala/io/getquill/ast/AstShowSpec.scala

```diff
@@ -505,4 +505,35 @@ class AstShowSpec extends Spec {
     (q.ast: Ast).show mustEqual
       """query[TestEntity].distinct"""
   }
+
+  "shows schema" - {
+    "table" - {
+      val q = quote {
+        query[TestEntity].schema(s => s.table("test"))
```

Comment:
I like `entity`. I've been avoiding because people correlate it to ORMs, but I'm fine with it if you are.